src/components/Header.js

```diff
@@ -0,0 +1,23 @@
+import styled from "styled-components";
+function Header() {
+  return (
+    <Head className="header">
+      <h4>Shop App</h4>
+    </Head>
+  );
+}
+const Head = styled.header`
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  border-bottom: 1px solid black;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  z-index: 0;
+  background-color: aliceblue;
+  margin-bottom: 200px;
```

Comment:
This value looks weird. 200px? Is this correct?

Reply:
with 200px i will reach a distance between the top window border and the further elements down to the Header. otherwise my search el. can be hidden under the Header. i will adapt it as soon i create next elements.